src/Template/Element/search_bar.ctp

```diff
@@ -28,17 +28,51 @@
 use Cake\Core\Configure;
 
 $searchQuery = h(str_replace('{{', '\{\{', json_encode($searchQuery)));
-?>
 
-<md-toolbar ng-controller="SearchBarController as ctrl" class="search_bar md-whiteframe-1dp md-primary">
-<?php
 if ($selectedLanguageFrom == null) {
     $selectedLanguageFrom = 'und';
 }
 
 if ($selectedLanguageTo == null) {
     $selectedLanguageTo = 'und';
 }
+?>
+
+<md-toolbar id="search-bar" ng-cloak>
+    <div class="md-toolbar-tools">
+    <?php
+    echo $this->Form->create('Sentence', [
+        'layout' => 'column',
+        'url' => ['controller' => 'sentences', 'action' => 'search'],
+        'type' => 'get',
+        'flex' => '',
+        'ng-cloak' => '',
+        'ng-controller' => 'SidenavController'
+    ]);
+    ?>
+
+    <div layout="row" layout-align="center center" flex>
+        <md-input-container class="md-accent" flex md-no-float>
+            <md-icon>search</md-icon>
+            <input name="query" 
+                accesskey="4" 
+                dir="auto" 
+                ng-model="ctrl.searchQuery" 
+                ng-init="ctrl.searchQuery = <?= $searchQuery ?>"
+                placeholder="<?= __('Search') ?>"/>
+        </md-input-container>
+        <md-button class="md-icon-button" ng-click="toggle('advanced-search')">
+            <md-icon>more_vert</md-icon>
+        </md-button>
```

Comment:
I’m not sure this "three-dots" button has enough visibility. Maybe this could be a button labelled "Narrow down" placed next to the search results title.

Besides, using small screens, I guess the only way to start a search after typing keywords is to use the Enter key. This might not be intuitive enough for many users. I think we should keep a "search" button.

Here is a draft of the two above suggestions using small screen:
```
≡          Tatoeba          🌐
[cat_dog________________] [🔎]
1234 results [Narrow down]
```

Of course the "Narrow down" button would allow more than that: modify results order, see more results and modify criteria. We could think about a different naming. But I think narrowing down the results currently really is the primary use case because most users will see too much results at first.